mannaggia.sh

```diff
@@ -32,7 +32,7 @@ shutdown=false
 off=false
 DELSTRING1="</FONT>"
 DELSTRING2="</b>"
-DEFPLAYER="mplayer -really-quiet -ao alsa"
+DEFPLAYER="mplayer -cache 1024 -"
```

Comment:
per il -really-quiet: l'output viene già soppresso nella riga 144
per -ao alsa: già ogni distribuzione sceglie il suo percorso di compatibilità e priorità in /etc/mplayer/mplayer.conf, quello di ubuntu 16.04 ha ao=pulse,alsa,sdl:aalib, quindi molti più driver audio in fallback.